docs/getting-started/getting-started-with-wdl.rst

```diff
@@ -208,11 +208,56 @@ the Dockstore command line and descriptor rather than just directly
 how to run your tool.
 
+Setting Up the Dockstore CLI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 The first thing I'll do is
 `setup the Dockstore CLI locally <https://dockstore.org/quick-start>`__.
 This will have me install all of the dependencies needed to run the
 Dockstore CLI on my local machine.
 
+Although not strictly required, it's a good idea to set a concurrent-job-limit when running locally. This is because Cromwell, when running locally, loses the ability to set compute resources within what is specified in a task's runtime attributes. As such, the local version of Cromwell sometimes uses too many resources when running on scattered tasks, causing the task to get sigkilled or for Docker to lock up. Of course, there is a tradeoff: If you set concurrent-job-limit, tests involving scattered tasks may execute slower as less instances of a scattered task will run in parallel.
+
+.. tip::  If a Docker lockup happens, you will notice tasks do not progress beyond WaitingForReturnCode and you will be temporarily unable to use Docker on your OS. Thankfully, this can be solved easily by restarting Docker on your machine.
+
+The easiest way to avoid these issues is to set up a Cromwell configuration file that sets concurrent-job-limit.
+
+1. Download `this template file <https://github.com/broadinstitute/cromwell/blob/develop/cromwell.example.backends/cromwell.examples.conf>`__ and name it ``.cromwell.conf``
+2. Uncomment ``#default = "LocalExample"`` in the ``backend`` section in order to override the default local Cromwell setup with what is in the configuration file.
+3. Under ``LocalExample``, under ``config``, uncomment the setting for ``concurrent-job-limit`` and set it to 1. Note that this is an integer, not a boolean; you could set it to 2 if you want to up to two jobs to run at once.  
+
+The relevent part of your configuration file should now look like this, if we exclude the other providers that come before LocalExample:
+
+::
+
+    backend {
+    # Override the default backend.
+    default = "LocalExample"
+
+    # The list of providers.
+    providers {
+       
+        # Define a new backend provider.
+        LocalExample {
+            # The actor that runs the backend. In this case, it's the Shared File System (SFS) ConfigBackend.
+            actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
+          
+            # The backend custom configuration.
+            config {
+                # Optional limits on the number of concurrent jobs
+                concurrent-job-limit = 1
+
+You may optionally edit ``root = "cromwell-executions"`` to something else if you wish to be certain that your configuration is getting used, as it will change the name of the executions folder. In any case, close that file and add the following line to ~/.dockstore/config
+
+::
+
+    cromwell-vm-options: -Dconfig.file=<path-to-your-cromwell-config>
```

Comment:
Does the path need to be absolute? If yes, then I suggest `<absolute-path-to...`

Reply:
Good spot. I think it can be relative, but mine is absolute and there is no reason not to make that the standard.